server/lib/search.js

```diff
@@ -178,7 +178,7 @@ export const searchCollectivesInDB = async (
   }
 
   if (tags?.length) {
-    searchedTags = `{${tags.map(tag => `"${tag}"`).join(',')}}`;
+    searchedTags = `{${tags.map(tag => `"${tag.toLowerCase()}"`).join(',')}}`;
```

Comment:
I don't get why we're changing this here?

Reply:
Sorry about this one, I was trying this out for a different issue and somehow accidentally checked in.